common/util.go

```diff
@@ -115,3 +118,18 @@ func GetBinlogRelativeDir(binlogDir string) string {
 	instanceID := filepath.Base(binlogDir)
 	return filepath.Join("backup", "instance", instanceID)
 }
+
+// GetAvailableFSSpace gets the free space of the mounted filesystem.
```

Comment:
As this is used only once, it may not be common. Let's keep it close to where it is used.

Reply:
Done.